### PR TITLE
Edit Wiki Style Guides

### PR DESCRIPTION
**Pull request by filippoweb3**
Refresh the Wiki contributor guides. More detailed information can be added on a future PR.

### CODE REVIEW

File: docs/general/contributing.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ Please do not try to pull request any marketing material as this will be rejecte
 :::
 
 Nonetheless, pull requests, discussions, and contributions from the community are encouraged. Active
-community members who demonstrate a record of good contributions may be given write access to the
+community members who demonstrate a record of good contributions may be given written access to the
```
Comment (profradha) [inline docs/general/contributing.md]: read/write access are GitHub terms. Can be kept as is. 
https://stackoverflow.com/questions/70538793/remote-write-access-to-repository-not-granted-fatal-unable-to-access

```suggestion
community members who demonstrate a record of good contributions may be given `write` access to the
```

File: docs/general/contributing.md [hunk 2]
```diff
@@ -68,19 +68,111 @@ There are a few basic ground-rules for contributors:
...
+:::
+
+The Polkadot Wiki is powered by [Docusaurus](https://docusaurus.io/). Docusaurus 3 was a major
+version upgrade that caused [MDX errors](https://docusaurus.io/docs/3.2.1/migration/v3). As a
+general rule, avoid using curly brackets and special characters. If you must use them, for example,
+to get the \<-\> symbol, you can escape the error with a `\`.
+
+A consequence of the Docusaurus 3 upgrade is the removal of the conditional rendering with Kusama
+and system chains. If you need to display text for different chains, use `Tabs` and `TabItem`
+formatting.
```
Comment (profradha) [inline docs/general/contributing.md]: It is okay to just say we moved away from conditional rendering. 

It was a custom built feature by us and it is possible to have conditional rendering with Docusaurus 3. My understanding was that it was not worth the effort, given the decision to remove Kusama guide. 

The code for conditional rendering is here:
https://github.com/w3f/polkadot-wiki/blob/master/scripts/injectPlugin.js

File: docs/general/contributing.md [hunk 3]
```diff
@@ -68,19 +68,111 @@ There are a few basic ground-rules for contributors:
...
+[the Wiki readme file](https://github.com/w3f/polkadot-wiki?tab=readme-ov-file#contributing-to-documentation).
+
+:::
+
+The Polkadot Wiki is powered by [Docusaurus](https://docusaurus.io/). Docusaurus 3 was a major
+version upgrade that caused [MDX errors](https://docusaurus.io/docs/3.2.1/migration/v3). As a
+general rule, avoid using curly brackets and special characters. If you must use them, for example,
+to get the \<-\> symbol, you can escape the error with a `\`.
+
+A consequence of the Docusaurus 3 upgrade is the removal of the conditional rendering with Kusama
```
Comment (profradha) [inline docs/general/contributing.md]:
```suggestion
Following the Docusaurus 3 upgrade, conditional rendering has been disabled for Kusama
```

File: docs/general/contributing.md [hunk 4]
```diff
@@ -68,19 +68,111 @@ There are a few basic ground-rules for contributors:
...
+
+To ensure consistency across pages, note the following general terms:
+
+- `relay chain` in text and `Relay chain` at the beginning of a sentence.
+- `parachain` in text and `Parachain` at the beginning of a sentence.
+
+The Polkadot Wiki has multiple moving parts. To ensure your contribution does not introduce
+compilation errors and that your page is properly rendered in production, we encourage you to load
+the Wiki locally before submitting a pull request for review. Loading the Wiki locally also adds the
+benefit of checking for broken links.
```
Comment (profradha) [inline docs/general/contributing.md]:
```suggestion
compilation errors and that your page is properly rendered in production, we encourage you to build and render
the Wiki on your local machine before submitting a pull request for review. Rendering the Wiki locally also adds the
benefit of checking for broken links.
```

**Comment by filippoweb3** [comment]
@DrW3RK thanks for the review. I implemented the changes and will keep updating this page while doing the reorg.

**Review by DrW3RK — APPROVED** [review]
Sounds good! Thanks for adding the Wiki style guidelines.

**Pull request merged**